keps/sig-network/2943-networkpolicy-status/kep.yaml

```diff
@@ -15,28 +15,23 @@ approvers:
   - "@thockin"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha 
+stage: 
 
 see-also:
   - "/keps/sig-network/2079-network-policy-port-range"
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.24"
+latest-milestone: 
```

Comment:
hm.. if we have to remove code then I think that counts as "work toward delivery of this KEP"?

Reply:
yes, I spoke with @palnabarun and he explained me this should be v1.27, that is when the feature was withdrawn. I'm changing here